Derrickkip/Ride-My-Way
# #158996647 Make GET /rides public

#### What does this PR do?
Make `GET /rides` public

#### Description of task to be completed?
* Move validation schemas to `schemas.py`
* Add RidesList endpoint
* Remove `@jwt_required` from `GET /rides`
* Update documentation

#### How to test this manually
* Clone the repo
* Create a .env file
* Create a test database and put the url in the .env
* Run the app `python server.py`
* Using postman navigate to `GET /rides`
* You should not be required to provide an authentication token

#### Relevant PT stories
[#158996647](https://www.pivotaltracker.com/story/show/158996647)
[#158986621](https://www.pivotaltracker.com/story/show/158986621)
